jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/utils/ClickAwayListenerTests.java

```diff
@@ -6,66 +6,6 @@
 
 import java.util.ArrayList;
 
-import static io.github.com.StaticSite.utilsClickAwayListenerDefaultPage;
-import static io.github.com.StaticSite.clickAwayListenerFrame;
-import static io.github.com.StaticSite.utilsClickAwayListenerPortalExamplePage;
-
 public class ClickAwayListenerTests extends TestsInit {
 
```

Comment:
Do you really want to delete this tests?

Reply:
It's from 2725-2 branch